Engine/Alphas/StatisticsInsightManagerExtension.cs

```diff
@@ -99,16 +104,26 @@ public void OnInsightClosed(InsightAnalysisContext context)
             var volume = _tradablePercentOfVolume * context.InitialValues.Volume;
 
             // value of the entering the trade in the account currency
-            var enterValue = volume * context.InitialValues.Price * context.InitialValues.QuoteCurrencyConversionRate;
+            var enterValue = context.InitialValues.Price * context.InitialValues.QuoteCurrencyConversionRate;
 
             // value of exiting the trade in the account currency
-            var exitValue = volume * context.CurrentValues.Price * context.CurrentValues.QuoteCurrencyConversionRate;
+            var exitValue = context.CurrentValues.Price * context.CurrentValues.QuoteCurrencyConversionRate;
 
             // total value delta between enter and exit values
             var insightValue = (int)context.Insight.Direction * (exitValue - enterValue);
 
-            context.Insight.EstimatedValue = insightValue;
-            Statistics.TotalAccumulatedEstimatedAlphaValue += insightValue;
+            var insightValueFactoredByTradableVolume = insightValue * volume;
+
+            context.Insight.EstimatedValue = insightValueFactoredByTradableVolume;
+            Statistics.TotalAccumulatedEstimatedAlphaValue += insightValueFactoredByTradableVolume;
+
+            // testing showed sometimes enter value is 0, example is ScheduledUniverseSelectionModelRegressionAlgorithm
```

Comment:
any investigation into why we sometimes get a zero here? what the underlying cause may be?

Reply:
The initial price is zero, I'll make a separate GH issue to investigate further 👍